cmd/katapult/main_test.go

```diff
@@ -23,6 +24,12 @@ func assertCobraCommand(t *testing.T, cmd *cobra.Command, errResult, want, stder
 	}
 	require.NoError(t, err)
 
-	assert.Equal(t, want, stdout.String())
+	if golden.Update() {
+		golden.Set(t, stdout.Bytes())
+	}
+	assert.Equal(t, string(golden.Get(t)), stdout.String())
 	assert.Equal(t, stderrResult, stderr.String())
 }
+
+const forMapTpl = `{{ range $key, $value := . }}{{ $key }}{{ $value }}{{ end }}
```

Comment:
What's this ?

Reply:
Testing the template. Using a string for the test would mean that you couldn't tell if item was echoed, but maps mean we can make sure that it is actually rendering the template.

Reply:
Hmm, yeah,  where's the test that actually uses it though ??

Reply:
`output_test.go` (Test_outputWrapper and Test_genericOutput_Text). I can't remember why it's in main_test though, shall I move it to that file?

Reply:
Looks like it's only used in `output_test.go`, so maybe we could move this const there? :)

Reply:
That would be preferable, otherwise it's very random and wtf